meta-genivi-dev/meta-genivi-dev/recipes-navigation/positioning/positioning_git.bb

```diff
@@ -61,31 +61,29 @@ DEPENDS_${PN}-enhpos = "${PN}-gnss ${PN}-sns"
 RDEPENDS_${PN}-enhpos-test = "${PN}-enhpos"
 DEPENDS_${PN}-enhpos-test = "${PN}-enhpos"
 
-do_configure() {
-    cd ${S} && cmake -DWITH_DLT=OFF -DWITH_GPSD=OFF -DWITH_REPLAYER=ON -DWITH_TESTS=ON -DWITH_IPHONE=OFF .
-}
-
-do_compile() {
-    cd ${S} && make
-}
+EXTRA_OECMAKE = "-DWITH_DLT=OFF -DWITH_GPSD=OFF -DWITH_REPLAYER=ON -DWITH_TESTS=ON -DWITH_IPHONE=OFF"
 
 do_install() {
     install -d ${D}/${bindir}
     install -d ${D}/${libdir}
     install -d ${D}${includedir}/${PN}
     install -d ${D}${datadir}/${PN}
-    install -m 755 ${S}/log-replayer/src/log-replayer ${D}/${bindir}
-    install -m 755 ${S}/log-replayer/test/test-log-replayer ${D}/${bindir}
-    install -m 644 ${S}/log-replayer/logs/*.log ${D}${datadir}/${PN}
-    install -m 755 ${S}/gnss-service/src/*.so ${D}/${libdir}
-    install -m 755 ${S}/gnss-service/test/gnss-service-client ${D}/${bindir}
-    install -m 755 ${S}/gnss-service/test/compliance-test/gnss-service-compliance-test ${D}/${bindir}
-    install -m 755 ${S}/sensors-service/src/*.so ${D}/${libdir}
-    install -m 755 ${S}/sensors-service/test/sensors-service-client ${D}/${bindir}
-    install -m 755 ${S}/enhanced-position-service/*/src/enhanced-position-service ${D}/${bindir}
-    install -m 755 ${S}/enhanced-position-service/*/test/enhanced-position-client ${D}/${bindir}
+
+    install -m 755 ${B}/log-replayer/src/log-replayer ${D}/${bindir}
+    install -m 755 ${B}/log-replayer/test/test-log-replayer ${D}/${bindir}
+    install -m 644 ${B}/log-replayer/logs/*.log ${D}${datadir}/${PN}
+
+    install -m 755 ${B}/gnss-service/src/*.so ${D}/${libdir}
+    install -m 755 ${B}/gnss-service/test/gnss-service-client ${D}/${bindir}
+    install -m 755 ${B}/gnss-service/test/compliance-test/gnss-service-compliance-test ${D}/${bindir}
+
+    install -m 755 ${B}/sensors-service/src/*.so ${D}/${libdir}
+    install -m 755 ${B}/sensors-service/test/sensors-service-client ${D}/${bindir}
+
+    install -m 755 ${B}/enhanced-position-service/*/src/enhanced-position-service ${D}/${bindir}
+    install -m 755 ${B}/enhanced-position-service/*/test/enhanced-position-client ${D}/${bindir}
     install -m 755 ${S}/enhanced-position-service/*/api/*.xml ${D}${datadir}/${PN}
-    install -m 755 ${S}/enhanced-position-service/dbus/api/*.h ${D}${includedir}/${PN}
+    install -m 755 ${B}/enhanced-position-service/dbus/api/*.h ${D}${includedir}/${PN}
```

Comment:
So, this {S} was a bug before?  If so, the commit comment ought to mention it.

Reply:
Nah, previously they ran "cmake .", meaning that everything ended up in ${S}, even build output. 

cmake.bbclass sets,

    B = "${WORKDIR}/build"

And that is where all build output is stored.